sourcegraph/sourcegraph
# Add compute live preview component

Closes https://github.com/sourcegraph/sourcegraph/issues/37966

[Related backend PR](https://github.com/sourcegraph/sourcegraph/pull/38261)

[Designs](https://www.figma.com/file/DqZHetdfqHSVzzWtyfjOmr/Compute-x-Insights%2C-RFC-696%2C-%2337923-%5BReady-fro-dev%5D?node-id=1964%3A18576)

This is simply a static rendering of the preview component. The backend pieces will be implemented here [Code Insights: Assemble compute-powered insight creation UI](https://github.com/sourcegraph/sourcegraph/issues/37965)

## Test plan

- See the fully rendered preview in storybook
- See the blurred preview by navigating to `insights/create/compute`
- Both should match designs

## App preview:

- [Web](https://sg-web-insights-add-compute-live-preview.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-fdoskhpwxu.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
